requirements/quality.txt

```diff
@@ -212,7 +194,7 @@ pytest==7.1.3
     #   -r requirements/test.txt
     #   pytest-cov
     #   pytest-django
-pytest-cov==3.0.0
+pytest-cov==4.0.0
```

Comment:
Dropped support for multi-processing: https://pytest-cov.readthedocs.io/en/latest/changelog.html#id1